test/QtiParsingTest.php

```diff
@@ -31,64 +31,93 @@
  *
  * @author Bertrand Chevrier, <taosupport@tudor.lu>
  * @package taoQTI
- 
+
  */
 class QtiParsingTest extends TaoPhpUnitTestRunner {
-	
+
 	protected $qtiService;
-	
+
 	/**
 	 * tests initialization
 	 */
-	public function setUp(){		
+	public function setUp(){
 		TaoPhpUnitTestRunner::initTest();
 		$this->qtiService = Service::singleton();
 	}
-	
+
 	/**
 	 * test qti file parsing: validation and loading in a non-persistant context
 	 */
 	public function testFileParsingQti2p1(){
-		
+
 		//check if wrong files are not validated correctly
 		foreach(glob(dirname(__FILE__).'/samples/wrong/*.*') as $file){
 			$qtiParser = new Parser($file);
 			$qtiParser->validate();
-			
+
 			$this->assertFalse($qtiParser->isValid());
 			$this->assertTrue(count($qtiParser->getErrors()) > 0);
 			$this->assertTrue(strlen($qtiParser->displayErrors()) > 0);
 		}
-		
+
         $files = array_merge(
                 glob(dirname(__FILE__).'/samples/xml/qtiv2p1/*.xml'), glob(dirname(__FILE__).'/samples/xml/qtiv2p1/rubricBlock/*.xml')
         );
-		//check if samples are loaded 
+		//check if samples are loaded
 		foreach($files as $file){
-            
+
 			$qtiParser = new Parser($file);
 			$qtiParser->validate();
-			
+
             if(!$qtiParser->isValid()){
                 echo $qtiParser->displayErrors();
             }
-			
+
 			$this->assertTrue($qtiParser->isValid());
-			
+
 			$item = $qtiParser->load();
 
             $this->assertInstanceOf('\\oat\\taoQtiItem\\model\\qti\\Item', $item);
-            
+
 		}
-        
+
 	}
-	
+    /**
+     * test if a correctResponse with CDATA works
+     * @author Thibault Milan <thibault.milan@vesperiagroup.com>
+     */
+    public function testFileParsingCDATA(){
+        $file = dirname(__FILE__).'/samples/xml/cdata/item.xml';
+
+        $qtiParser = new Parser($file);
+        $qtiParser->validate();
+
+        if(!$qtiParser->isValid()){
+            echo $qtiParser->displayErrors();
```

Comment:
I was just reusing the code just before my test :D But thanks, I'll change that just in a sec